### PR TITLE
fixing openjdk_version in community.yaml

### PR DESCRIPTION
**Pull request by coreydaley**
It looks like this was probably a copy/paste error from the official.yaml file.

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ Path: community/infinispan/templates/infinispan-persistent.json
...
+Source URL: [https://raw.githubusercontent.com/jboss-container-images/openjdk/develop/templates/community-image-streams.json](https://raw.githubusercontent.com/jboss-container-images/openjdk/develop/templates/community-image-streams.json )  
 Docs: [https://access.redhat.com/documentation/en-us/red_hat_jboss_middleware_for_openshift/3/paged/red_hat_java_s2i_for_openshift/](https://access.redhat.com/documentation/en-us/red_hat_jboss_middleware_for_openshift/3/paged/red_hat_java_s2i_for_openshift/)  
 Path: community/java/imagestreams/java-ubi8.json  
 ## templates
 ### openjdk-web-basic-s2i
-Source URL: [https://raw.githubusercontent.com/jboss-container-images/openjdk/release/templates/openjdk-web-basic-s2i.json](https://raw.githubusercontent.com/jboss-container-images/openjdk/release/templates/openjdk-web-basic-s2i.json )  
+Source URL: [https://raw.githubusercontent.com/jboss-container-images/openjdk/develop/templates/openjdk-web-basic-s2i.json](https://raw.githubusercontent.com/jboss-container-images/openjdk/develop/templates/openjdk-web-basic-s2i.json )
```
Comment (adambkaplan) [inline community/README.md:41]: We need to verify if the OpenJDK team wants the `release` or `develop` branch to be added to OpenShift.

ping @jmtd

Comment (coreydaley) [reply community/README.md:41]: These are just docs and an index that are going away in the rewrite anyways.

Comment (adambkaplan) [reply community/README.md:41]: nm - see the diff now.

Comment (jmtd) [reply community/README.md:41]: @adambkaplan I thought it was a good idea to use develop for OKD - which will follow the latest improvements we make to image streams, and release for product, which is only updated when a product image change is made.

**Pull request merged**